packages/entity-invalidator/src/index.ts

```diff
@@ -149,72 +149,30 @@ function delay(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
-async function invalidateAndWait({
-  cloudfront,
-  DistributionId,
-  Paths,
-}: {
-  cloudfront: AWS.CloudFront,
-  DistributionId: string,
-  Paths: {
-    Quantity: number;
-    Items: string[]
-  }
-}) {
-  const invalidationResponse = await cloudfront
-    .createInvalidation({
-      DistributionId,
-      InvalidationBatch: {
-        CallerReference: `${uuidv4()}`,
-        Paths,
-      },
-    })
-    .promise();
-
-  if (invalidationResponse?.Invalidation) {
-    const invalidationId = invalidationResponse.Invalidation.Id;
-
-    // Check the status of the invalidation batch periodically
-    let invalidationStatus = await getInvalidationStatus(
-      cloudfront,
-      invalidationId
-    );
-    let totalWait = 0;
-    while (invalidationStatus !== "Completed") {
-      console.log("Invalidation in progress. Waiting for completion...");
-      totalWait += INVALIDATION_WAIT
-      await delay(INVALIDATION_WAIT); // Wait for 10 seconds before checking the status again
-      invalidationStatus = await getInvalidationStatus(
-        cloudfront,
-        invalidationId
-      );
-
-      if (totalWait > INVALIDATION_WAIT_LIMIT) {
-        throw new Error("Exceeded invalidation wait limit")
-      }
-    }
+function getPaths(entity: KeyToAsset): string[] {
+  const v1 = `${DOMAIN}/v1/${entity.address}`;
+  const v2 = `${DOMAIN}/v2/hotspot/${entity.address}`;
+  if ((entity.entityKeyStr?.length || 0) >= 200) {
+    return [v1, v2];
   }
+
+  return [`${DOMAIN}/${entity.entityKeyStr!}`, v1, v2];
 }
 
-async function getInvalidationStatus(cloudfront: AWS.CloudFront, invalidationId: string) {
-  const invalidationResponse = await cloudfront
-    .getInvalidation({
-      DistributionId: CLOUDFRONT_DISTRIBUTION,
-      Id: invalidationId,
-    })
-    .promise();
+async function invalidate(arg: any): Promise<void> {
+  try {
+    const body = JSON.stringify(arg);
 
-  return invalidationResponse?.Invalidation?.Status;
-}
+    const response = await fetch(CLOUDFLARE_API_URL, { method: 'POST', headers: CLOUDFLARE_HEADERS, body: body });
+    const data = await response.json();
 
-function getPaths(entity: KeyToAsset): string[] {
-  const v1 = `/v1/${entity.address}`;
-  const v2 = `/v2/hotspot/${entity.address}`;
-  if ((entity.entityKeyStr?.length || 0) >= 200) {
-    return [v1, v2];
-  }
+    console.log(data);
```

Comment:
rm?

Reply:
The console.log is for us track invalidation responses from the Cloudflare API but I can remove it if need be

Reply:
All good, just though it was leftover and didn't serve a purpose